magpie/constants.py

```diff
@@ -146,6 +154,11 @@ def _get_default_log_level():
 MAGPIE_CONTEXT_PERMISSION = "MAGPIE_CONTEXT_USER"  # path user must be itself, MAGPIE_LOGGED_USER or unauthenticated
 MAGPIE_LOGGED_USER = "current"
 MAGPIE_DEFAULT_PROVIDER = "ziggurat"
+MAGPIE_NETWORK_TOKEN_NAME = "magpie_token"  # nosec: B105
+MAGPIE_NETWORK_PROVIDER = "magpie_network"
+MAGPIE_NETWORK_NAME_PREFIX = "anonymous_network_"
```

Comment:
Not blocker, but since `anonymous` is simply the default not-logged-in name, and could be set to a different value, this could lose its meaning for nodes changing it. 

Are there assumptions that each node will use `anonymous` when referring to this concept?